packages/types/src/phase0/validator.ts

```diff
@@ -0,0 +1,76 @@
+import {ByteViews, ContainerNodeStructType, ValueOfFields} from "@chainsafe/ssz";
+import * as primitiveSsz from "../primitive/sszTypes.js";
+
+const {Boolean, Bytes32, UintNum64, BLSPubkey, EpochInf} = primitiveSsz;
+
+// this is to work with uint32, see https://github.com/ChainSafe/ssz/blob/ssz-v0.15.1/packages/ssz/src/type/uint.ts
+const NUMBER_2_POW_32 = 2 ** 32;
+
+/**
```

Comment:
multiline comment instead of jsdoc

```suggestion
/*
```